docs/command_list/newt_mfg.rst

```diff
@@ -87,16 +87,25 @@ The ``rb_blinky_rsa`` package needs a pkg.yml file. In addition it is needs a mf
 .. code-block:: console
 
     $  more mfgs/rb_blinky_rsa/mfg.yml
-    mfg.bootloader: 'targets/rb_boot'
-    mfg.images:
-        - 'targets/rb_blinky'
+    mfg.bsp: "@apache-mynewt-core/hw/bsp/rb-nano2"
+    mfg.targets:
+        - rb_boot:
+          name: "targets/rb_boot"
+          area: FLASH_AREA_BOOTLOADER
+          offset: 0x0
+        - rb_blinky:
+          name: "targets/rb_blinky"
+          area: FLASH_AREA_IMAGE_0
+          offset: 0x0
+    mfg.meta:
+        area: FLASH_AREA_BOOTLOADER
 
 Build the bootloader and app images.
 
 .. code-block:: console
 
-    $ newt build rb_boot
-    $ newt create-image rb_blinky 0.0.1
+    $ newt build rb_boot && newt create-image rb_boot 0.0.1
```

Comment:
Assuming `rb_boot` is a boot loader target, that should be `newt build rb_boot` (rather than `newt create-image rb_boot 0.0.1`).

`create-image` would also work, but it creates an unnecessary `.img` file with an image header (and it requires you to specify a version number that doesn't get included anywhere).

Reply:
@ccollins476ad, thanks for your feedback. I removed it.